src/elements/head.pug

```diff
@@ -0,0 +1,10 @@
+head
+	title Parliamentary Digital Service
+	meta(name="viewport", content="width=device-width, initial-scale=1.0")
+	meta(http-equiv="X-UA-Compatible", content="IE=edge")
+	if index
+		link(rel="stylesheet", href="_public/stylesheets/main.css")
+		link(rel="icon", type="image/x-icon", href="_public/images/favicon.ico")
+	else
+		link(rel="stylesheet", href="../../stylesheets/main.css")
+		link(rel="icon", type="image/x-icon", href="../../images/favicon.ico")
```

Comment:
What if the stylesheets _aren't_ two levels down, or aren't in public? What're the use cases that this would be different?